pyconcz/common/admin.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib import admin
+from django.template import loader
+from django import forms
+
+from .models import Phase, PhaseValue
+
+
+class ValueInline(admin.TabularInline):
+    model = PhaseValue
+    fields = ['name', 'enabled', 'active_since']
+    list_display = ['name', 'enabled', 'active_since']
+    list_editable = ['name', 'enabled', 'active_since']
+
```

Comment:
I'd add `extra = 0` (so no empty lines by default), but might be a matter of taste

Reply:
But then you can't add new values (unless you turn on Javascript).

Reply:
I put in `extra = 1`.

Reply:
I thought you bring that up. 😀

Reply:
I'm stuck in the time when *progressive enhancement* was the buzzword. I try to make things work on my Symbian-based phone from 360 BC – and if some new technology makes things easier to use, then sure, layer it on top.

In my view, if Javascript manages to start up and get to the point where it can modify the DOM, *then* it should be free to remove the extra lines and replace them with its `+` button. Sadly, Django doesn't seem to do it that way :(

Reply:
It's OK and _progressive enhancement_ is a good thing.